pymc3/step_methods/hmc/nuts.py

```diff
@@ -90,7 +91,8 @@ class NUTS(BaseHMC):
     def __init__(self, vars=None, Emax=1000, target_accept=0.8,
                  gamma=0.05, k=0.75, t0=10, adapt_step_size=True,
                  max_treedepth=10, on_error='summary', **kwargs):
-        R"""
+        R"""Set up the No-U-Turn sampler.
```

Comment:
It looks like most of the class doc string is documenting the `sampler_stats` return object -- I hadn't seen that there is an `Attribute` section for class docstrings though, which is probably where the sampler stats should go...